src/main/java/pl/tomaszdziurko/jvm_bloggers/view/panels/CustomPageLinkModel.java

```diff
@@ -0,0 +1,51 @@
+package pl.tomaszdziurko.jvm_bloggers.view.panels;
+
+import lombok.RequiredArgsConstructor;
+
+import org.apache.wicket.markup.html.navigation.paging.IPageable;
+import org.apache.wicket.model.IModel;
+
+import java.io.Serializable;
+
+@RequiredArgsConstructor
+public class CustomPageLinkModel implements IModel<String>, Serializable {
```

Comment:
You can extend [`AbstractReadOnlyModel`](https://ci.apache.org/projects/wicket/apidocs/6.x/org/apache/wicket/model/AbstractReadOnlyModel.html) instead implementing `IModel` directly. This way you can get rid of empty [`setObject()`](https://github.com/tdziurko/jvm-bloggers/pull/167/files#diff-a2d7860c43e7bed9e9803f81433c6847R23) and [`detach()`](https://github.com/tdziurko/jvm-bloggers/pull/167/files#diff-a2d7860c43e7bed9e9803f81433c6847R23) method implementations.